.github/scripts/commons.mjs

```diff
@@ -8,17 +8,20 @@ export const config = {
   testAssetsDir: `./test-assets/`,
 }
 
+export const branchName = process.env.BRANCH_NAME || 'main';
```

Comment:
```suggestion
export const branchName = process.env.BRANCH_NAME ?? 'main';
```
I usually prefer nullish coalescence to avoid weird effects with falsy values.